src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -155,7 +155,19 @@ public SelectExpandNode(SelectExpandClause selectExpandClause, IEdmStructuredTyp
         /// <summary>
         /// Gets the list of EDM navigation properties to be expanded in the response.
         /// </summary>
-        public IDictionary<IEdmNavigationProperty, SelectExpandClause> ExpandedNavigationProperties { get; private set; }
+        public IDictionary<IEdmNavigationProperty, SelectExpandClause> ExpandedNavigationProperties
```

Comment:
>ExpandedNavigationProperties [](start = 71, length = 28)

we really don't want users to call this since it will create a new dictionary each time, and it won't work like they might think it will (i.e., they can't add to the dictionary to affect the set of expanded navigation properties associated with the selecteexpandnode.  We should deprecate this in favor of the new (renamed) property.